src/consumeroffset/kafka_offset_collect_test.go

```diff
@@ -220,13 +215,93 @@ func TestCollectOffsetsForConsumerGroup(t *testing.T) { // nolint: funlen
 					// this entity only for topicOne that has member clientID
 					assert.Equal(t, testClientID, entity.Metrics[0].Metrics["clientID"])
 					assert.Equal(t, topicOne, entity.Metrics[0].Metrics["topic"])
+				default:
+					assert.Fail(t, "not expected")
 				}
 				assert.NotEmpty(t, entity)
 			}
 		})
 	}
 }
 
+func TestNoActiveConsumersForConsumerGroup(t *testing.T) { // nolint: funlen
+	// MemberAssignment mock created as in sarama's consumer_group_member_test.go
+	members := map[string]*sarama.GroupMemberDescription{
+		testClientID: {
+			ClientId:       testClientID,
+			ClientHost:     "a-host",
+			MemberMetadata: nil,
+			MemberAssignment: []byte{
+				0, 0, // Version
+				0, 0, 0, 0, // Topic array length
+				0, 0, 0, 3, 0x01, 0x02, 0x03, // Userdata
+			},
+		},
+	}
```

Comment:
Would it make sense to add a comment highlighting there are no active costumers here?